integration-tests/src/liquidation.rs

```diff
@@ -53,8 +53,13 @@ pub fn supply(mm_pool: EvmAddress, user: EvmAddress, asset: EvmAddress, amount:
 		.write(0u32)
 		.build();
 
-	let (res, value) = Executor::<Runtime>::call(context, data, U256::zero(), 500_000);
-	assert_eq!(res, Succeed(Returned), "{:?}", hex::encode(value));
+	let call_result = Executor::<Runtime>::call(context, data, U256::zero(), 500_000);
```

Comment:
Please don't do unnecessary changes like these. This only creates more code to review without any added value. 
I saw it in multiple files. You don't have to revert it but for the future